data/mods/ssb/scripts.ts

```diff
@@ -23,8 +23,10 @@ export const BattleScripts: ModdedBattleScriptsData = {
 		this.runEvent('AfterMega', pokemon);
 
 		// Kaiju Bunny gains the fairy type when mega evolving
-		if (pokemon.name === 'Kaiju Bunny' && !pokemon.illusion) this.add('-start', pokemon, 'typeadd', 'Fairy');
-
+		if (pokemon.name === 'Kaiju Bunny' && !pokemon.illusion) {
+			this.add('-start', pokemon, 'typeadd', 'Fairy');
+			pokemon.types = ['Normal', 'Fairy'];
+		}
```

Comment:
```suggestion
		if (pokemon.name === 'Kaiju Bunny' && !pokemon.illusion) this.add('-start', pokemon, 'typechange', 'Normal/Fairy');
```
This isn't correct. it should be typechange, not typeadd. You also don't need to mutate the types in here.